splash/src/main/res/layout/activity_splash.xml

```diff
@@ -1,20 +1,94 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please put `xmlns:android` declaration onto its own line.

Reply:
Auto-indentation will reset the change and send it back to the first line with parent tag. Leaving it as it is. We could update our code style settings if you really want it.

Reply:
I do want it, but it's not important now. We'll need to revisit indentation standards at some point anyway.